src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets

```diff
@@ -333,13 +336,26 @@ Copyright (c) .NET Foundation. All rights reserved.
       <Output TaskParameter="AssetsWithTargetPath" ItemName="_WasmJsModuleCandidatesForBuildWithTargetPath" />
     </ComputeStaticWebAssetsTargetPaths>
```

Comment:
I think you can also get rid of this one if you aren't using the target path for anything.

Reply:
I see now that the target path is still being used for the initializers. You can get rid of that too by passing the initializers and their endpoints https://github.com/dotnet/runtime/pull/103755/files#diff-acd6e3e7ab8ee0e70d5b2be89c136e2928953c651e800d35188591c7f3ca9541R347

Reply:
That'll give you the fingerprinted endpoints for the initializers

Reply:
```xml
    <FilterStaticWebAssetEndpoints Condition="'@(_WasmJsModuleCandidatesForBuild)' != ''"
      Endpoints="@(StaticWebAssetEndpoint)"
      Assets="@(_WasmJsModuleCandidatesForBuild)"
      Filters=""
    >
      <Output TaskParameter="FilteredEndpoints" ItemName="_WasmJsModuleCandidatesForBuildEndpoint" />
    </FilterStaticWebAssetEndpoints>
```